scripts/src/public/services/offers.ts

```diff
@@ -72,12 +72,10 @@ async function filterOffers(userId: string, app: Application | undefined, logger
 	const offerCounts = await Order.countAllByOffer(userId);
 	const contents = await offerContents.getAllContents();
 	let availableTranslations: OfferTranslation[] = [];
-	let language: string | null = null;
+	let language: string | false = false;
```

Comment:
why false?

Reply:
That's what the `acceptsLanguages` function returns when no supported language is found.